packages/1xdev-users/CHANGELOG.md

```diff
@@ -2,78 +2,3 @@
 
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
```

Comment:
You are changing other package's files, rather than your package. These belong to `1xdev-users`